### DISCUSSION TITLE
Raytracing Resolution

### DISCUSSION THREAD

**Discussion by CJRAWR** [praydog/REFramework#697] [General Questions]
Hi I saw this framework is able to modify some of the graphics, I was wondering if the scope of this project could include increasing the resolution of raytracing in the Next Gen Games, https://youtu.be/Nu61HMZPEXg, this video shows off the crux of the problem, even at highest settings, the effect is quarter resolution of the game. This is made worse because DLSS uses smaller resolutions, so having a native version of all these effects would be a nice upgrade to future proof the game! Thanks for listening!

**Comment by TommyCreo21** [comment]
That would be a great option to have.
